### PR TITLE
Changes to make get_the_content compatible with PHP 7.2+

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#137371](http://central.tri.be/issues/137371)
---
Prevent warnings from happening, ~I will need to include a changelog here.~

Depends on https://github.com/moderntribe/tribe-common/pull/1211

New Video: https://i.bordoni.me/WnuAnn57

### CODE REVIEW

**Comment by bordoni** [comment]
Actually on the beta I might not have that particular changelog, I will leave it for the merge into `release/B19.15`

**Comment by bordoni** [comment]
Tests are failing due to a false alert. Happens every PR that runs later in the day.

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -2792,7 +2792,7 @@ public function googleCalendarLink( $post = null ) {
 			// Fetch the
 			$location = trim( $this->fullAddressString( $post->ID ) );
 
-			$event_details = apply_filters( 'the_content', get_the_content( $post->ID ) );
+			$event_details = apply_filters( 'the_content', get_the_content( null, false, $post->ID ) );
```
Comment (borkweb) [inline src/Tribe/Main.php]: This should be updated to `tribe_get_the_content()`

Comment (bordoni) [inline src/Tribe/Main.php]:
```suggestion
			$event_details = tribe_get_the_content( null, false, $post->ID ) );
```

**Pull request merged**